config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  # get 'root/index'
+  # get 'greetings/random'
+  # # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    resources :greetings, only: :index
+   
+   end
+  #  greetings#index
+   root 'root#index'
+  # # Defines the root path route ("/")
+  # get 'greeting', to: '/api/greetings#index'
+  # # root "articles#index"
```

Comment:
Kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 🙏